### ISSUE TITLE
Click of anchor tag with jQuery AJAX does not work

### ISSUE THREAD

**Issue by jtanul** [MachinePublishers/jBrowserDriver#127]
Hello, I have HTML like this:

```
<a class="btn xt-btn-sm btn-primary" id="edit18" href="#" onclick=" $(this).addClass('disabled'); jQuery.ajax({type:'POST',data:'personId=18', url:'/myapp/person/edit/',success:function(data,textStatus){jQuery('#mainBody').html(data);},error:function(XMLHttpRequest,textStatus,errorThrown){popupUnexpectedError();}, complete:function(xhr,textStatus){  $(this).removeClass('disabled');  }});" title="Edit Record">
    <i class="glyphicon glyphicon-edit"></i>&nbsp;
</a>
```

And I invoke like this

`driver.findElement(By.id("edit18")).click();`

But the ajax is not fired.  This works ok with Firefox web driver and also htmlunitdriver. Any clue?

**Comment by hollingsworthd** [comment]
This seems to be working based on a simplified test case I tried. It may have been fixed as part of the v14.0.0 release, specifically https://github.com/MachinePublishers/jBrowserDriver/issues/110 

Can you confirm, is this still broken? If so, I will try to reproduce this using HTML more similar to yours, maybe it's related to the ajax post or something.

**Comment by hollingsworthd** [comment]
Between #110 and other changes I am guessing this may be fixed. Please re-open if not.
